flink-ml-iteration/src/main/java/org/apache/flink/iteration/operator/InputOperator.java

```diff
@@ -53,13 +53,14 @@ public void processElement(StreamRecord<T> streamRecord) throws Exception {
 
     @Override
     public void endInput() throws Exception {
-        if (insertMaxEpochWatermark) {
-            reusable.replace(
-                    IterationRecord.newEpochWatermark(
-                            Integer.MAX_VALUE,
-                            OperatorUtils.getUniqueSenderId(
-                                    getOperatorID(), getRuntimeContext().getIndexOfThisSubtask())));
-            output.collect(reusable);
-        }
+        //        if (insertMaxEpochWatermark) {
```

Comment:
I think it might be better to remove the no use code. WDYT?

Reply:
I'll remove this code~